### ISSUE TITLE
Init message shows `yarn start` when installed using npm

### ISSUE THREAD

**Issue by chancancode** [ember-cli/ember-cli#8859]
```
$ ember --version
ember-cli: 3.13.0
node: 12.8.1
os: linux x64
$ ember new super-rentals -b @ember/octane-app-blueprint
installing octane-app-blueprint
  create .editorconfig
  create .ember-cli.js
  create .eslintignore
  create .eslintrc.js
  create .template-lintrc.js
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app/app.js
  create app/index.html
  create app/resolver.js
  create app/router.js
  create app/styles/app.css
  create app/templates/application.hbs
  create config/environment.js
  create config/optional-features.json
  create config/targets.js
  create ember-cli-build.js
  create .gitignore
  create jsconfig.json
  create package.json
  create public/robots.txt
  create testem.js
  create tests/index.html
  create tests/test-helper.js

Installing packages... This might take a couple of minutes.
npm: Installing dependencies ...
npm: Installed dependencies

Initializing git repository.
Git: successfully initialized.

Successfully created project super-rentals.
We suggest you that you get started by typing:

  $ cd super-rentals
  $ yarn start

Happy coding!
```

The last `$ yarn start` should be `$ npm start`

**Comment by rwjblue** [comment]
I believe that this was introduced by https://github.com/ember-cli/ember-cli/pull/8588. I believe we need to update the message to say `npm start` when the `--yarn` flag was not used.

@dcyriller - Do you think you might have some time to take a look at this? If you do, we'd need to target the `release` branch with a PR (since it affects 3.13.0+ at this point).

**Comment by dcyriller** [comment]
Here is a fix: https://github.com/ember-cli/ember-cli/pull/8862

**Comment by dcyriller** [comment]
Sorry, I'm a yarn user, in case you didn't notice 😉.

**Comment by rwjblue** [comment]
Fixed by #8862

**Comment by rwjblue** [comment]
Thank you again @dcyriller!
